### PR TITLE
Issue-456 input configuration should not be altered as it breaks unix…

### PR DESCRIPTION
**Pull request by marcuslinke**
… socket support

Fix issue #456 in master branch. Relates to https://github.com/docker-java/docker-java/pull/457

### CODE REVIEW

File: src/main/java/com/github/dockerjava/jaxrs/DockerCmdExecFactoryImpl.java [hunk 1]
```diff
@@ -209,7 +209,14 @@ public void init(DockerClientConfig dockerClientConfig) {
 
         client = clientBuilder.build();
 
-        baseResource = client.target(originalUri.toString()).path(dockerClientConfig.getApiVersion().asWebPathPart());
+        baseResource = client.target(sanitizeUrl(originalUri).toString()).path(dockerClientConfig.getApiVersion().asWebPathPart());
+    }
+
+    private URI sanitizeUrl(URI originalUri) {
```
Comment (KostyaSha) [inline src/main/java/com/github/dockerjava/jaxrs/DockerCmdExecFactoryImpl.java]: Please provide some javadoc to get idea why it critical.

Comment (marcuslinke) [inline src/main/java/com/github/dockerjava/jaxrs/DockerCmdExecFactoryImpl.java]: Sorry, don't know what you mean.

Comment (KostyaSha) [inline src/main/java/com/github/dockerjava/jaxrs/DockerCmdExecFactoryImpl.java]: well, ok, ignore.

**Comment by KostyaSha** [comment]
Let's try.

**Pull request merged**